src/raw.ts

```diff
@@ -1061,13 +1061,35 @@ export class RawRemarkable implements RawRemarkableApi {
     const raw = await this.getText(hash);
     const loaded = JSON.parse(raw) as unknown;
 
-    // jtd can't verify non-discriminated unions, in this case, we have fileType
-    // defined or not. As a result, we try each, and concatenate the errors at the end
+    // Normalize unopened document fields (documents uploaded but never opened)
```

Comment:
instead of trying to normalize here, I'd rather just change the definition to indicate that pages can be null, and textAlignment can be the empty string